### PR TITLE
Add Grab

### PR DESCRIPTION
**Pull request by LitoMore**
**Issue:** #6085 
**Alexa rank:** 10,985

### Checklist
- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Source from https://en.wikipedia.org/wiki/File:Grab_(application)_logo.svg  
`#00B14F`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3694,6 +3694,11 @@
                 ]
             }
         },
+        {
+            "title": "Grab",
+            "hex": "1AAE48",
+            "source": "https://en.wikipedia.org/wiki/Grab_(company)"
```
Comment (ericcornelissen) [inline _data/simple-icons.json:3700]: As per [our source guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), this should be: 

```suggestion
            "source": "https://en.wikipedia.org/wiki/File:Grab_(application)_logo.svg)"
```

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
SVG looks spot on compared to the source. Just to be clear (e.g. in case this is questioned in the future), where did you source the hex value from? Other than that, just one request for a change and this is good to merge, nice work @LitoMore 👍

**Comment by LitoMore** [comment]
@ericcornelissen OK. Let me check the hex value with my former colleague at Grab.

**Comment by LitoMore** [comment]
@ericcornelissen Color updated. The color is picked from the Grab code base.

**Comment by LitoMore** [comment]
@ericcornelissen Color updated.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -3694,6 +3694,11 @@
                 ]
             }
         },
+        {
+            "title": "Grab",
+            "hex": "00B14F",
+            "source": "https://en.wikipedia.org/wiki/File:Grab_(application)_logo.svg)"
```
Comment (ericcornelissen) [inline _data/simple-icons.json:3700]: My apologies, something went wrong in my previous suggestion :sweat_smile: 

```suggestion
            "source": "https://en.wikipedia.org/wiki/File:Grab_(application)_logo.svg"
```

**Comment by ericcornelissen** [comment]
@LitoMore, do you have any indication of where we might find this color publicly? If not, I'll leave this for another @simple-icons/maintainers to got their OK and merge it in :slightly_smiling_face:

**Comment by LitoMore** [comment]
@ericcornelissen We could find their brand color `#00B14F` everywhere in the package.

For example the background of Download App button:

**Review by ericcornelissen — APPROVED** [review]

**Comment by ericcornelissen** [comment]
Thank you very much for your contribution @LitoMore :confetti_ball:

**Pull request merged**